backend/database.py

```diff
@@ -1,26 +1,226 @@
-from flask import Flask
-from flask_sqlalchemy import SQLAlchemy
-
-app = Flask(__name__)
-app.config["SQLALCHEMY_DATABASE_URI"] = "sqlite:///lab8.sqlite" 
-db = SQLAlchemy(app)
-
-class User(db.Model):
-    u_name = db.Column(db.String, primary_key=True)
-    u_password = db.Column(db.String, nullable=False)
-    u_type = db.Column(db.String, nullable=False)
-
-class Classes(db.Model):
-    c_classkey = db.Column(db.Integer, nullable=False, primary_key=True)
-    c_classname = db.Column(db.String, nullable=False)
-    c_teacher = db.Column(db.String, nullable=False)
-    c_time = db.Column(db.String, nullable=False)
-    c_seats = db.Column(db.Integer, nullable=False)
-
-class Students_in_Classes(db.Model):
-    sc_classkey = db.Column(db.Integer, nullable=False, primary_key=True)
-    sc_name = db.Column(db.String, nullable=False)
-    sc_grade = db.Column(db.Integer, nullable=False)
-
-db.create_all()
+from ast import get_docstring
+import sqlite3
+
+def get_all_of_role(c, role):
+    command = '''
+    SELECT * FROM Users
+    WHERE u_role = \"''' + str(role) +'''\" ;
```

Comment:
We should be using "qmark" style to craft queries. We should never use string manipulations. That leaves us open to SQL injection attacks. See: https://docs.python.org/3/library/sqlite3.html